### PR TITLE
Reorder/Reparent sections in the Matroska RFC

### PR DESCRIPTION
**Pull request by robUx4**
The sections are currently hard to follow. The flow of information doesn't go logically.
I reordered a few sections to make things a little better (more improvements are probably needed).

This is needed before #355 where some parts of the Block sections need some work. It's better if they are in a finalize place/file in the document.

The EBML contraints from the Schema were duplicate and less clear than the manually created section.

**NO CONTENT WAS CHANGED,** only text moved and `#` levels edited.

### CODE REVIEW

**Comment by robUx4** [comment]
I uploaded the [old TXT layout](https://github.com/robUx4/matroska-specification/blob/old-layout/draft-ietf-cellar-matroska-08.txt) (before this PR) and the [new TXT layout](https://github.com/robUx4/matroska-specification/blob/new-layout/draft-ietf-cellar-matroska-08.txt) (after this PR) so it's easier to [compare them](https://github.com/robUx4/matroska-specification/pull/1/files).

The summary/index is the part to focus on, to see where the section were/are. The text within these sections is untouched.
One of the main changes is that the elements sections in the summary are less details (not every single elements of SeekHead/Info/etc are listed). They are not needed in the summary/index.

**Review by mcr — APPROVED** [review]

**Review by retokromer — APPROVED** [review]

**Comment by SpencerDawkins** [comment]
I made this comment on the mailing list, but then realize that I should have put it in github - and half of this is about the reordering topic. 

I see that Michael has already approved the PR in https://github.com/robUx4/matroska-specification/branches, but I did have one quick question, after a quick look through the table of contents. 

       9.1.4.  Tracks Element  . . . . . . . . . . . . . . . . . . .  36
       9.1.5.  Cues Element  . . . . . . . . . . . . . . . . . . . . 102

So, there's a 66-page gap between 9.1.4 and 9.1.5. The specification has plenty of 9.1.4.n subsections, but they don't show up in the TOC, because the "tocDepth" attribute isn't specified in the XML, and defaults to "3", so they won't be easily spotted unless you know what to search for in the text. 

Perhaps you could consider adding "tocDepth=4" to 

<rfc version="3" ipr="trust200902" docName="draft-ietf-cellar-matroska-08" sortRefs="true" submissionType="IETF" category="info" xml:lang="en" xmlns:xi="http://www.w3.org/2001/XInclude" consensus="true">

and see if the result is manageable?

**Comment by SpencerDawkins** [comment]
While cross-checking the XML, I noticed that

<title abbrev="Matroska Format">Matroska Media Container Format Specifications</title><seriesInfo value="draft-ietf-cellar-matroska-08" stream="IETF" status="informational" name="Internet-Draft"></seriesInfo>

says that the specification is "informational", but the CELLAR charter at https://datatracker.ietf.org/wg/cellar/about/ says that the Matroska specifications would be Standards Track, so status="std". 

Would it make sense to make that change at the same time?

Best,

Spencer

**Comment by robUx4** [comment]
> ```
>    9.1.4.  Tracks Element  . . . . . . . . . . . . . . . . . . .  36
>    9.1.5.  Cues Element  . . . . . . . . . . . . . . . . . . . . 102
> ```
> 
> So, there's a 66-page gap between 9.1.4 and 9.1.5. The specification has plenty of 9.1.4.n subsections, but they don't show up in the TOC, because the "tocDepth" attribute isn't specified in the XML, and defaults to "3", so they won't be easily spotted unless you know what to search for in the text.
> 
> Perhaps you could consider adding "tocDepth=4" to

I thought the index looked a bit nicer without all the small details, but a 66 pages gap is indeed not nice at all.

Looking at the previous system we had before this PR it wasn't much better though
```
     9.4.  Tracks Element  . . . . . . . . . . . . . . . . . . . . .  44
       9.4.1.  TrackEntry Element  . . . . . . . . . . . . . . . . .  44
     9.5.  Cues Element  . . . . . . . . . . . . . . . . . . . . . . 109
       9.5.1.  CuePoint Element  . . . . . . . . . . . . . . . . . . 109
```

So I'd rather fix it in a separate PR to discuss that specific issue.

**Comment by robUx4** [comment]
> While cross-checking the XML, I noticed that
> <title abbrev="Matroska Format">Matroska Media Container Format Specifications</title>
> 
> says that the specification is "informational", but the CELLAR charter at https://datatracker.ietf.org/wg/cellar/about/ says that the Matroska specifications would be Standards Track, so status="std".
> 
> Would it make sense to make that change at the same time?

Addressed in #512

**Comment by SpencerDawkins** [comment]
@robUx4 - both your responses sound perfect. Thanks for considering my comments.

**Comment by SpencerDawkins** [comment]
Merged during interim meeting
